frontend/src/components/SideBar.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import {ReactComponent as HomeIcon} from '../assets/icons/home.svg';
+import {ReactComponent as SetlistIcon} from '../assets/icons/list.svg';
+import {ReactComponent as TrackIcon} from '../assets/icons/record-thicc.svg';
+import {ReactComponent as ArtistIcon} from '../assets/icons/mic-thicc.svg';
+
+
+const IconClassName = 'icon';
+
+const Sidebar: React.FC = () => {
```

Comment:
Yep, this is good. You can also do `import { FC } from 'react';` and then this line would be `const Sidebar: FC = () => {`

Reply:
great, i'll update those now